src/util/sss_ldb.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __SSS_LDB_H__
+#define __SSS_LDB_H__
+
+#include "config.h"
+
+#ifdef LDB_MODULES_PATH_OVERRIDE
+void sss_ldb_init_modules_path(void);
```

Comment:
Do we really need a new file?
Could this be a static function in `src/confdb/confdb.c` directly?

Reply:
Hi,

I think is good idea, At first I wanted to separate it to make it clearer to identify. But I think it made the code unnecessarily heavy.

Best Regards